mxcubeweb/core/models/configmodels.py

```diff
@@ -160,3 +160,64 @@ class AppConfigModel(BaseModel):
     server: FlaskConfigModel
     mxcube: MXCUBEAppConfigModel
     sso: SSOConfigModel | None
+
+
+class AdapterResourceHandlerConfigModel(BaseModel):
+    """
+    Used to define  which adapter properties and methods that are
```

Comment:
The configuration model for a adapter resource that outlines how a resource is defined.